README.md

```diff
@@ -426,7 +416,7 @@ $ oc get route vote-ui --template='http://{{.spec.host}}'
 ```
 
 
-If you want to re-run the pipeline again, you can use the following short-hand command to rerun the last pipelinerun again that uses the same pipeline resources and service account used in the previous pipeline run:
+If you want to re-run the pipeline again, you can use the following short-hand command to rerun the last pipelinerun again that uses the same pipeline resources, workspaces, params and service account used in the previous pipeline run:
```

Comment:
I suggest to omit `pipeline resources` as we don't use them (althought the statement is techically correct).